scripts/libs.js

```diff
@@ -0,0 +1,72 @@
+/* eslint-disable import/no-unresolved */
+/* eslint-disable import/no-dynamic-require */
+/* eslint-disable no-console */
+const fs = require('fs');
+const os = require('os');
+
+const args = process.argv.slice(2);
+const jsonPath = args[0];
+const mdPath = args[1];
+const licenseCheckerPath = args[2];
+const licenseCrawlerPath = args[3];
+const licensesCheckerData = require(licenseCheckerPath);
+const licensesCrawlerData = require(licenseCrawlerPath);
+const line = os.EOL;
+const doubleLine = `${os.EOL}${os.EOL}`;
+
+const disableLink = text => text.replace(/\./g, '<i></i>.');
```

Comment:
This is a kind of hack. Markdown file automatically detects URL addreses and converts them to links. So if it finds a library whose name is similar to the URL address (Socket.io) it automatically converts this name to link, but often this is a wrong link. Because we want links only where we planned them. I looked for a solution but all the solutions were about some hacks. When we add this `<i></i>` before each dot this string isn't recognized as a link...

Reply:
So add a comment about what it does, so someone doesn't have to guess in next month what this function does.